js/widgets/page.dialog.js

```diff
@@ -33,63 +33,83 @@
 
 return $.widget( "mobile.page", $.mobile.page, {
 	options: {
+		classes: {
+			"ui-page-dialog-close-button":
+			"ui-button ui-corner-all ui-button-icon-only",
+			"ui-page-dialog-close-button-icon": "ui-icon-delete ui-icon",
+			"ui-page-dialog-contain": "ui-overlay-shadow ui-corner-all"
+		},
 
 		// Accepts left, right and none
 		closeBtn: "left",
 		closeBtnText: "Close",
 		overlayTheme: "a",
-		corners: true,
 		dialog: false
 	},
 
 	_create: function() {
+		this._ui = {};
```

Comment:
Im not a fan of the `_ui` name space here it does not actually describe what it is. We also have a mixed convention on the use of `_` for private. We have gone back and forth on this. I think i would prefer to go with the ui convention that the `_ ` is only for methods we have never had public properties in any widget im aware of so there is no need to mark them private. They are also not effected by the widget bridge which makes private methods not directly callable through the bridge.

I think i would prefer `this.dialog = {};` here personally